<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewImage</class>
 <widget class="QDialog" name="ViewImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="push_back">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_image">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>image</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Error!</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
